modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -0,0 +1,58 @@
+
+*	Before this update, the OpenShift Logging dashboard displayed an empty CPU graph because of a reference to an invalid metric. With this update, the correct data point has been selected, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-2026[LOG-2026])
+
+*	Before this update, the Fluentd container image included builder tools that were unnecessary at run time. This update removes those tools from the image.(link:https://issues.redhat.com/browse/LOG-1927[LOG-1927])
+
+*	Before this update, a name change of the deployed collector in the 5.3 release caused the logging collector to generate the `FluentdNodeDown` alert. This update resolves the issue by fixing the job name for the Prometheus alert. 	(link:https://issues.redhat.com/browse/LOG-1918[LOG-1918])
+
+*	Before this update, the log collector was collecting its own logs due to a refactoring of the component name change. This could lead to a potential feedback loop of the collector processing its own log that might result in memory and log message size issues. This update resolves the issue by excluding the collector logs from the collection. (link:https://issues.redhat.com/browse/LOG-1774[LOG-1774])
+
+* Before this update, Elasticsearch generated the error "Unable to create PersistentVolumeClaim due to forbidden: exceeded quota: infra-storage-quota." if the PVC already existed. With this update, Elasticsearch checks for existing PVCs, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-2131[LOG-2131])
+
+* Before this update, Elasticsearch was unable to return to the ready state when the `elasticsearch-signing `secret was removed. With this update, Elasticsearch is able to go back to the ready state after that secret is removed.	(link:https://issues.redhat.com/browse/LOG-2171[LOG-2171])
+
+* Before this update, the change of the path from which the collector reads container logs caused the collector to forward some records to the wrong indices. With this update, the collector now uses the correct configuration to resolve the issue.	(link:https://issues.redhat.com/browse/LOG-2160[LOG-2160])
+
+* Before this update, clusters with a large number of namespaces caused Elasticsearch to stop serving requests because the list of namespaces reached the maximum header size limit. With this update, headers only include a list of namespace names, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-1899[LOG-1899])
+
+* Before this update, the OpenShift Logging dashboard use to show number of shards 'x' times bigger than actual value when Elasticsearch has 'x' nodes. This was because it was printing all primary shards for each ES pod and processing sum on it while the output is always for the whole ES cluster. With this update, the calculation has been corrected.	(link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])
```

Comment:
```suggestion
* Before this update, the OpenShift Logging dashboard showed the number of shards 'x' times bigger than actual value when Elasticsearch has 'x' nodes. This was because it was printing all primary shards for each ES pod and processing sum on it, while the output is always for the whole ES cluster. With this update, the calculation has been corrected.	(link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])
```